comp/remote-config/rcserviceha/rcservicehaimpl/rcserviceha.go

```diff
@@ -63,20 +63,36 @@ func newHaRemoteConfigServiceOptional(deps dependencies) optional.Option[rcservi
 // newHaRemoteConfigServiceOptional creates and configures a new service that receives remote config updates from the configured DD failover DC
 func newHaRemoteConfigService(deps dependencies) (rcserviceha.Component, error) {
 	apiKey := configUtils.SanitizeAPIKey(deps.Cfg.GetString("ha.api_key"))
-	baseRawURL, err := configUtils.GetHAEndpoint(deps.Cfg, "https://config.", "ha.rc_dd_url")
+	baseRawURL, err := configUtils.GetHAEndpoint(deps.Cfg, "https://config.", "remote_configuration.ha.rc_dd_url")
```

Comment:
<!-- Thoughts represent an idea that popped up from reviewing. These comments are non-blocking by nature, but they are extremely valuable and can lead to more focused initiatives and mentoring opportunities.  -->
**💭 Thought:** ‏Kinda wonder if we don't want to put every ha param under ha, meaning that this'd become `ha.remote_configuration.rc_dd_url`

Reply:
oooo okay yeah I like that better. Will do 👍